docs/developer_docs.md

```diff
@@ -43,6 +43,8 @@ This plugin supports sending text messages to the Virtual Agent through Mattermo
 
     - [Pre-defined Conversation Flows/Topics](https://docs.servicenow.com/bundle/quebec-now-intelligence/page/administer/virtual-agent/reference/prebuilt-topics-ITSM.html)
     - [Additional Plugins](https://docs.servicenow.com/bundle/sandiego-servicenow-platform/page/administer/virtual-agent/reference/additional-va-plugins.html)
+    - Sometimes a conversation flow might not work because the user do not have access to some tables or APIs which are being used in that flow (You can see the errors in "All > System Log > Errors"). In such cases you have to manually provide the access in "All > Application Cross-Scope Access".
```

Comment:
```suggestion
    - Sometimes a conversation flow might not work because the user does not have access to some tables or APIs which are being used in that flow (You can see the errors in "All > System Log > Errors"). In such cases, you have to manually provide the access in "All > Application Cross-Scope Access".
```